### ISSUE TITLE
IOS 12 Progressive Web App Responsive resizing bug

### ISSUE THREAD

**Issue by Sonargroup** [alvarotrigo/fullPage.js#3488]
### window.innerHeight is returned incorrectly from the getWindowHeight function when the fullpage.js site is running as a PWA on iPad

I am unable to include a link as you need to add a website (configured as a PWA) to the home screen on ipad in order to see the bug.

```
function getWindowHeight(){
    return  'innerHeight' in window ? window.innerHeight : document.documentElement.offsetHeight;
}
```

### Steps to reproduce it
1. Open the website in safari on ipad and change the orientation of the screen, the sections and slides will resize to fill the screen.
2. Click 'add to home screen' and install the website as a PWA
3. Open the PWA and change the screen orientation, the sections and slides will not resize in height correctly.

### Current fix
I modified the getWindowHeight so that it uses document.documentElement.offsetHeight if it is set.

```
function getWindowHeight(){
    return  document.documentElement.offsetHeight ? document.documentElement.offsetHeight : window.innerHeight;
}
```

For some reason this solution required that I also trigger a rebuild after the initial call to $('#fullpage').fullpage({...}); I suspect that document.documentElement.offsetHeight is not set or not correct when the page first initialises (even though all code is being run inside a DOM ready function).

```
setTimeout(function() { fullpage_api.reBuild(); }, 200);
```

I can't say I spent much time looking into the code base but my solution seems to work well, maybe there is a better way to do it and hopefully this might help someone coming across a similar issue.

### Versions 
iOS 11+, fullpage.js 3.0.3, interlockedSlides.js 0.0.5, jQuery 3..3.1

**Comment by alvarotrigo** [comment]
Thanks for reporting it!
I'll have to check it with more time!

**Comment by alvarotrigo** [comment]
I was able to reproduce the issue. 
I'll take a look at it these days!

**Comment by alvarotrigo** [comment]
This seems to be a bug in Webkit browsers:
https://bugs.webkit.org/show_bug.cgi?id=170595

Still investigating what can be the correct "hacky solution".

**Comment by alvarotrigo** [comment]
This issue seems to have been fixed at the same time as [this bug](https://github.com/alvarotrigo/fullPage.js/issues/3336) and will get merged in 3.0.6.
